scripts/jenkins/cloud/ansible/deploy-crowbar.yml

```diff
@@ -47,6 +47,13 @@
             - reboot_after_deploy
             - not update_after_deploy
 
+        - include_role:
```

Comment:
Just one more thing: if you're not fully targeting the "unified" CI deploying Crowbar in the engineering cloud, could you please remove this task ? By itself, it's not harmful, but I think this belongs together with the other changes required to make this work with the unified CI.